pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -248,6 +248,9 @@ pyramid_oereb:
   # The method used to return the municipality logos.
   get_municipality_method: pyramid_oereb.standard.hook_methods.get_municipality
 
+  # Flag to print or not the canton logo
```

Comment:
This must be in the print configuration section (only for mapfish print configuration options)